### PR TITLE
Feature/settle charges

### PR DESCRIPTION
**Pull request by jonatasmaxi**
If approved thir PR will add SettleCharges method to Subscription model

### CODE REVIEW

File: PagarMe.Tests/PagarMeTestFixture.cs [hunk 1]
```diff
@@ -91,6 +91,19 @@ public static Plan CreateTestPlan()
 			};
 		}
 
+        public static Plan CreateBoletoTestPlan()
```
Comment (murilohns) [inline PagarMe.Tests/PagarMeTestFixture.cs]: alinhar a identação

File: PagarMe.Tests/SubscriptionTests.cs [hunk 2]
```diff
@@ -80,5 +80,52 @@ public void CancelSubscription ()
 
 			Assert.AreEqual (subscription.Status, SubscriptionStatus.Canceled);
 		}
+        [Test]
+        public void SettleChargeWithoutParameters()
```
Comment (murilohns) [inline PagarMe.Tests/SubscriptionTests.cs]: alinhar a identação

File: PagarMe.Tests/SubscriptionTests.cs [hunk 3]
```diff
@@ -80,5 +80,52 @@ public void CancelSubscription ()
...
+
+            Assert.AreEqual(subscription.Status, SubscriptionStatus.Paid);
+        } 
+        [Test]
+        public void SettleChargeWithParameters()
+        {
+            var plan = CreateBoletoTestPlan();
+            plan.Save();
+
+            Assert.AreNotEqual(plan.Id, 0);
```
Comment (murilohns) [inline PagarMe.Tests/SubscriptionTests.cs]: você precisa dessa linha?

File: PagarMe.Shared/Model/Subscription.cs [hunk 4]
```diff
@@ -146,6 +146,12 @@ public PaymentMethod PaymentMethod
             set { SetAttribute("payment_method", value); }
         }
 
+        public int[] SettledCharges
+        {
+            get { return GetAttribute<int[]>("settled_charges"); }
+
```
Comment (murilohns) [inline PagarMe.Shared/Model/Subscription.cs]: extra line?

File: PagarMe.Tests/SubscriptionTests.cs (hunk 3 above)
Comment (jonatasmaxi) [reply PagarMe.Tests/SubscriptionTests.cs]: Checking if plan was created @murilohns

**Pull request closed without merging**